### PR TITLE
Support DW-Conv2D in QAT when quantizing per weights per channel

### PR DESCRIPTION
**Pull request by elad-c**
Fix bug with weight quantization per output channel when channel axis isn't the last axis

### CODE REVIEW

**Comment by haihabi** [comment]
Please add some unit testing for this case.

**Review by reuvenperetz — APPROVED** [review]

**Comment by elad-c** [comment]
Do not merge before **fix_weight_quantization** is merged

**Pull request merged**
